+sw_tests/+unit_tests/unittest_spinw_spec2MDHisto.m

```diff
@@ -0,0 +1,39 @@
+classdef unittest_spinw_spec2MDHisto < sw_tests.unit_tests.unittest_super
+    properties
+    end
+    methods (Test)
+        function test_1_0_0(testCase)
+            q0 = [0 0 0];
+            qdir = [1 0 0];
+            spec = sw_egrid(spinwave(sw_model('triAF', 1), {q0 q0+qdir nsteps}));
```

Comment:
More importantly when I add the missing `end` and run the tests they fail because `nsteps` isn't defined!